### ISSUE TITLE
Turn on Twig devMode by default

### ISSUE THREAD

**Issue by derekderaps** [kalamuna/kalastatic#465]
I'm not sure if Twig.js implements `devMode`, but if it does, let's make sure it's turned on by default. It does things like throws exceptions when referencing variables that don't exist, etc. Things that are turned off in production.

**Comment by RobLoach** [comment]
Yes please... Made this issue to start the discussion over in the Twig.js queue http://bit.ly/2qqRnBa
